use-symfony-reproducer.md

```diff
@@ -0,0 +1,172 @@
+# Using a Symfony Reproducer
+
+***TIP
+This is part 2 of a 2-part _Reproducer_ series. If you haven't already, check
+out [Part 1](https://symfonycasts.com/blog/symfony-reproducer) first.
+***
+
+Ok, so you've created a reproducer app for a bug you've found in Symfony. For
+this article, we'll assume the reproducer you've created in part 1 is located
+at `~/my-reproducer`.
+
+## Forking the Symfony Repository
+
+The first thing we'll do is _fork_ the [`symfony/symfony`](https://github.com/symfony/symfony)
+repository on GitHub. This will give us our own copy of the Symfony repository
+where we can work.
+
+1. Visit https://github.com/symfony/symfony and click the _Fork_
+   button at the top of the page.
+2. Choose your personal GitHub account as the owner and keep the default name
+   of `symfony`.
+3. Leave all the other options as is and click _Create Fork_.
+4. You should now be on https://github.com/your-username/symfony.
+5. Wait a few seconds for the fork to be created.
+6. Click the green _Code_ dropdown and under the _SSH_ tab,
+   copy the URL to your clipboard.
+
+## Cloning your Fork
+
+Let's get the repo down to your local workspace:
+
+```bash
+cd ~
+git clone <your-fork-url-from-clipboard>
+cd symfony
+```
+
+Now we're going to add the original Symfony repo as an _upstream_ remote. Why?
+So we can easily pull in the latest Symfony updates and stay fresh!
+
+```bash
+git remote add upstream git@github.com:symfony/symfony.git
+```
+
+Let's grab the latest changes from Symfony (you'll want to run this command
+whenever Symfony gets an update, so it's good to get used to it).
+
+```bash
+git fetch upstream
+```
+
+## Use your Reproducer to Fix the Bug
+
+In Part 1, you picked the Symfony version for your reproducer. Now it's time
+to match that version in our forked Symfony repo. Let's check it out:
+
+```bash
+git checkout 7.1 # or any other version you picked for your reproducer
```

Comment:
Does this create that branch locally? I always use with `-b`, so I may be ignorant on what's happening

Reply:
It does but I can show the long form:

```
git checkout -b 7.1 upstream/7.1
```